### PR TITLE
Keep /raw PMs from showing up in main chat

### PR DESCRIPTION
**Pull request by bumbadadabum**

### CODE REVIEW

File: js/client-chat.js [hunk 1]
```diff
@@ -1145,7 +1145,7 @@
 			if (this.$chatFrame.scrollTop() + 60 >= this.$chat.height() - this.$chatFrame.height()) {
 				autoscroll = true;
 			}
-			this.addChat(user, message, pm);
+			if (!message.startsWith('/raw')) this.addChat(user, message, pm);
```
Comment (panpawn) [inline js/client-chat.js]: This should probably be done in ``this.addChat``

**Comment by Zarel** [comment]
`/html` tho

**Comment by bumbadadabum** [comment]
Is that ever sent over PM?

**Comment by Zarel** [comment]
Yes. The difference is that

> Zarel: `/raw <i>foo</i>`

will become 

> *foo*

while

> Zarel: `/html <i>foo</i>`

will become

> Zarel: *foo*

**Comment by bumbadadabum** [comment]
Guess I'll add that to the PR then

**Comment by Zarel** [comment]
Poke. I hope you didn't forget about this.

**Comment by bumbadadabum** [comment]
No I just haven't had the time, not home all day today. Will update when
I'm home.

**Pull request merged**